code/game/machinery/mass_driver.dm

```diff
@@ -96,106 +107,131 @@
 	icon_state = "mass_driver_frame"
 	density = FALSE
 	anchored = FALSE
-	var/build = 0
+	var/buildstage = DRIVER_FRAME
+
+/obj/machinery/mass_driver_frame/examine(mob/user)
+	. = ..()
+	switch(buildstage)
+		if(DRIVER_FRAME)
+			. += "<span class='notice'>Its <i>unbolted</i> from the [get_turf(src)] and could be <b>welded</b> down.</span>"
+		if(DRIVER_BOLTS)
+			. += "<span class='notice'>It could be securely <i>welded</i> to the [get_turf(src)] or could be <b>unbolted</b>.</span>"
+		if(DRIVER_WELDED)
+			. += "<span class='notice'>Its could be <i>wired</i> or <b>sliced</b> away from [get_turf(src)].</span>"
+		if(DRIVER_WIRED)
+			. += "<span class='notice'>Its missing its <i>grille</i> and is <b>wired</b>.</span>"
+		if(DRIVER_GRILLE)
+			. += "<span class='notice'>Its maintenance panel is <i>screwed</i> open and its <b>grille</b> is installed.</span>"
```

Comment:
```suggestion
		if(DRIVER_FRAME)
			. += "<span class='notice'>It's <i>unbolted</i> from [get_turf(src)] and could be <b>welded</b> down.</span>"
		if(DRIVER_BOLTS)
			. += "<span class='notice'>It could be securely <i>welded</i> to the [get_turf(src)] or could be <b>unbolted</b>.</span>"
		if(DRIVER_WELDED)
			. += "<span class='notice'>It could be <i>wired</i> or <b>sliced</b> away from [get_turf(src)].</span>"
		if(DRIVER_WIRED)
			. += "<span class='notice'>It's missing its <i>grille</i> and is <b>wired</b>.</span>"
		if(DRIVER_GRILLE)
			. += "<span class='notice'>Its maintenance panel is <i>screwed</i> open and its <b>grille</b> is installed.</span>"
```